rocAL/rocAL_pybind/examples/rocAL_api_external_source_example.py

```diff
@@ -0,0 +1,85 @@
+from random import shuffle
+from amd.rocal.pipeline import Pipeline
+from amd.rocal.plugin.pytorch import ROCALClassificationIterator
+import amd.rocal.fn as fn
+import amd.rocal.types as types
+import os
+
+def main():
+    batch_size = 3
+    data_dir = os.environ["ROCAL_DATA_PATH"] + "/coco/coco_10_img/train_10images_2017/"
+    device = "cpu"
+
+    try:
+        path = "OUTPUT_IMAGES_PYTHON/NEW_API/EXTERNAL_SOURCE_READER/"
+        isExist = os.path.exists(path)
+        if not isExist:
+            os.makedirs(path)
+    except OSError as error:
+        print(error)
+
+    #image is expected as a tensor
```

Comment:
What does this line signifies here?
    #image is expected as a tensor

Reply:
@swetha097 Please check this

Reply:
Renamed it to image_dump()
@shobana-mcw  Please confirm